composer.json

```diff
@@ -15,7 +15,7 @@
         }
     ],
     "require": {
-        "php": "^7.2",
+        "php": "^7.2|^8.0",
```

Comment:
Single pipe is not recommended and deprecated, you should use double pipe here.
```suggestion
        "php": "^7.2 || ^8.0",
```

It was in documentation before, but can't find where, so here is an issue about this https://github.com/composer/composer/issues/6755

Reply:
TIL, thank you @Korbeil :)